templates/labelVideo.html

```diff
@@ -20,11 +20,8 @@
       href="https://fonts.googleapis.com/icon?family=Material+Icons">
   <link rel="stylesheet"
       href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap">
-  <link rel="stylesheet" type="text/css"
-      href="https://storage.googleapis.com/{{project_id}}/static/css/styles.css?ignoreCache={{time_time}}">
-  <script type="text/javascript"
-      src="https://storage.googleapis.com/{{project_id}}/compiled/js/fmltc.js?ignoreCache={{time_time}}">
-  </script>
+  <link rel="stylesheet" type="text/css" href="static/css/styles.css">
```

Comment:
Does this work for quick changes to styles.css? I've had troubles with browsers caching the css and solved that by adding ?ignoreCache={{time_time}} to the url.

Reply:
I usually just do ctrl-f5 to flush the cache when I know I've changed css.  For ftc-scoring, when we push assets to the cloud we use a webpack cache busting technique to ensure that teams get the changes.  I'll look at that in more detail as we go.